vue-frontend/src/components/contact/ContactForm.vue

```diff
@@ -258,13 +260,39 @@ export default {
       showErrorMessagePopup: false,
       showLoader: false,
       contactType: 1,
+      hasRecaptchaRendered: false,
     };
   },
   components: {
     FontAwesomeIcon,
     CalendlyIframe,
   },
   methods: {
+    onloadCallback() {
+      grecaptcha.enterprise.ready(function () {
+        grecaptcha.enterprise
+          .execute("SITE_KEY", {
+            action: "login",
+          })
+          .then(function (token) {
+            console.log(token);
```

Comment:
Are you getting token?

Reply:
yes

Reply:
Just pass it to API and verify it using enterprise API

Reply:
Achha okay i try